packages/php-wasm/compile/php/Dockerfile

```
@@ -81,6 +88,31 @@ RUN if [[ "${PHP_VERSION:0:1}" -eq "7" && "${PHP_VERSION:2:1}" -le "3" ]]; then
 		cp /root/builds/libsqlite3/\<7\.4/sqlite_config0.m4 /root/php-src/ext/sqlite3/config0.m4; \
 	fi;
 
+# Patch OPcache config.m4 file if needed
+RUN if [ "$WITH_OPCACHE" = "yes" ]; \
+	then \
+		# Ensure mmap() shared memory is recognized, even if not true.
+		# This will #declare HAVE_SHM_MMAP_ANON 1 in php_config.h
+		if [[ "${PHP_VERSION:0:1}" -eq "8" && "${PHP_VERSION:2:1}" -eq "4" ]]; then \
+			# Name changed in 8.4. See https://github.com/php/php-src/commit/ca55603d8a27b4e1f9c12cf72acd0fd39a6ac059
+			/root/replace.sh 's/php_cv_shm_mmap_anon=no/php_cv_shm_mmap_anon=yes/g' /root/php-src/ext/opcache/config.m4; \
+		else \
+			/root/replace.sh 's/have_shm_mmap_anon=no/have_shm_mmap_anon=yes/g' /root/php-src/ext/opcache/config.m4; \
+			# PHP bug: PHP_NEW_EXTENSION do not read $ext_shared from the config.m4 file.
+			# https://github.com/php/php-src/blob/PHP-8.3.0/ext/opcache/config.m4#L313
```

Can we make this more of a permalink by referencing a specific commit?
```suggestion
			# https://github.com/php/php-src/blob/f35a22adba8a7ddf709c6cc98ecb57ba4dd8fe8c/ext/opcache/config.m4#L313
```

---

Added, thanks.